metadata-ingestion/src/datahub/ingestion/source/usage/bigquery_usage.py

```diff
@@ -462,10 +469,23 @@ def from_entry(cls, entry: AuditLogEntry) -> "QueryEvent":
             timestamp=entry.timestamp,
             actor_email=entry.payload["authenticationInfo"]["principalEmail"],
             query=job_query_conf["query"],
-        )
-        # jobName
-        query_event.jobName = _job_name_ref(
-            job.get("jobName", {}).get("projectId"), job.get("jobName", {}).get("jobId")
+            job_name=_job_name_ref(
```

Comment:
curious why you went from more defensive style `x.get('attr',{})` to more fragile `x['attr']`. Is it guaranteed that we find these columns? What if permissions don't allow you to see these columns.

Reply:
Thanks, you are right, I will fix it.